Project-14-Project-management-system/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "project-management-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
```

@sknepali Remove `^` from all dependencies and give reason for it.

---

The caret (^) symbol indicates a "compatible" update. It allows for updates that do not include breaking changes. 
```suggestion
    "react": "^18.2.0",
```

It signifies that the project can use any version greater than or equal to 18.2.0 but less than the next major version.